massa-execution-worker/src/execution.rs

```diff
@@ -1832,7 +1840,14 @@ impl ExecutionState {
             .final_state
             .read()
             .get_pos_state()
-            .get_address_deferred_credits(address);
+            .get_deferred_credits_range(.., Some(address))
+            .credits;
+        let res_final: BTreeMap<Slot, Amount> = res_final
```

Comment:
I think we could merge those 2 let bindings into one like this:
```rust
        let res_final: BTreeMap<Slot, Amount> = self
            .final_state
            .read()
            .get_pos_state()
            .get_deferred_credits_range(.., Some(address))
            .credits
            .iter()
            .filter_map(|(slot, addr_amount)| {
                addr_amount.get(address).map(|amount| (*slot, *amount))
            })
            .collect();
```

Reply:
Done